### PR TITLE
Add support for Pyup

### LINKED ISSUE #2149: [Icon Request] Add support for Pyup
- [x] I'm sure this issue is not a _duplicate_.
- [x] I'm submitting an icon request:
  - Type: `filename`
  - Icon Name: `pyup`
  - Sample original Icon: [logo](https://www.brandeps.com/logo/P/Pyup-01)
  - Filenames: `.pyup.yml`

- More info: [GitHub Project](https://github.com/pyupio), [pyup configuration](https://pyup.io/docs/bot/config/)

### PR DESCRIPTION
**Pull request by KingDarBoja**
_**Fixes #2149**_

**Changes proposed:**

- [x] Add
- [ ] Delete
- [ ] Fix
- [ ] Prepare

### CODE REVIEW

File: src/iconsManifest/supportedExtensions.ts [hunk 1]
```diff
@@ -2758,6 +2758,14 @@ export const extensions: IFileCollection = {
       languages: [languages.python],
       format: FileFormat.svg,
     },
+    {
+      icon: 'pyup',
+      extensions: ['.pyup'],
+      filenamesGlob: ['.pyup'],
```
Comment (JimiC) [inline src/iconsManifest/supportedExtensions.ts]: Don't use `*Glob` for just one filename. `*Glob` was designed to reduce lengthy declarations.

Comment (robertohuertasm) [reply src/iconsManifest/supportedExtensions.ts]: Changed!

File: src/iconsManifest/supportedExtensions.ts [hunk 2]
```diff
@@ -2771,6 +2771,12 @@ export const extensions: IFileCollection = {
       languages: [languages.python],
       format: FileFormat.svg,
     },
+    {
+      icon: 'pyup',
+      extensions: ['.pyup.yml'],
```
Comment (JimiC) [inline src/iconsManifest/supportedExtensions.ts]: `.pyup` is also valid.

Comment (JimiC) [reply src/iconsManifest/supportedExtensions.ts]: `pyup` `pyup.yml`

Comment (robertohuertasm) [reply src/iconsManifest/supportedExtensions.ts]: done!

**Pull request merged**